src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs

```diff
@@ -21,6 +21,9 @@ public static class VirtualNetworkGatewaySkuTier
         public const string HighPerformance = "HighPerformance";
         public const string Standard = "Standard";
         public const string UltraPerformance = "UltraPerformance";
+        public const string VpnGw1 = "VpnGw1";
```

Comment:
@ritwikbasu in order to publish this SDK, it has to be a different version that is already not published to nuget.
But also, if currently your SDK that is published on nuget as 10.0, but you end up publishing this particular with 9.1, that will create confusion.
this version should be 10.1.0